pkg/network/setup/netstat_test.go

```diff
@@ -463,6 +463,53 @@ var _ = Describe("netstat", func() {
 		})
 	})
 
+	Describe("primary interface position in spec", func() {
+		const (
+			prNetworkName   = "primary"
+			prMAC           = "1C:CE:C0:01:BE:E7"
+			secNetworkName1 = "secondary1"
+			secMAC1         = "1C:CE:C0:01:BE:E9"
+			secNetworkName2 = "secondary2"
+			secMAC2         = "1C:CE:C0:01:BE:E8"
+		)
+
+		BeforeEach(func() {
+			setup.Vmi.Spec.Networks = append(setup.Vmi.Spec.Networks, newVMISpecPodNetwork(prNetworkName), newVMISpecMultusNetwork(secNetworkName1))
+		})
+
+		DescribeTable("verify primary interface is always first in Status.Interfaces list", func(vmiSpecIfaces []v1.Interface, domainIfaces []api.Interface, statusIfaces []v1.VirtualMachineInstanceNetworkInterface) {
+			setup.Vmi.Spec.Domain.Devices.Interfaces = append(setup.Vmi.Spec.Domain.Devices.Interfaces, vmiSpecIfaces...)
+			setup.Domain.Spec.Devices.Interfaces = append(setup.Domain.Spec.Devices.Interfaces, domainIfaces...)
+
+			setup.NetStat.UpdateStatus(setup.Vmi, setup.Domain)
+
+			Expect(setup.Vmi.Status.Interfaces).To(Equal(statusIfaces))
+		},
+			Entry("primary interface defined first in spec",
+				[]v1.Interface{newVMISpecIfaceWithMasqueradeBinding(prNetworkName),
+					newVMISpecIfaceWithBridgeBinding(secNetworkName1)},
```

Comment:
It is challenging to read this in terms of format.
Can we have one of these options? :
```
[]v1.Interface{newVMISpecIfaceWithMasqueradeBinding(prNetworkName), newVMISpecIfaceWithBridgeBinding(secNetworkName1)},
```
OR
```
[]v1.Interface{
    newVMISpecIfaceWithMasqueradeBinding(prNetworkName),
    newVMISpecIfaceWithBridgeBinding(secNetworkName1),
},
```

Reply:
Test code changed